src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -51,7 +51,6 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 
     <!-- Trimmer defaults -->
-    <PublishTrimmed Condition="'$(PublishTrimmed)' == ''">true</PublishTrimmed>
     <TrimMode Condition="'$(TrimMode)' == ''">link</TrimMode>
```

Comment:
What do you think about moving the other 2 "Trimmer" properties as well? It feels odd to split them.

Reply:
I'm fine with doing that if @pranavkm approves that move too. Looks like those won't really get used until we actually call ILlink on a target so no issues with ordering with that one, but I agree that it makes sense to keep all together. Is it ok to make this change too @pranavkm @captainsafia?

Reply:
Yup should be fine. I'm trying to recall why they were in the targets file as opposed to the props file in the first place, and my likely guess is that this was based on 3.2 where trimming was only enabled in Release builds. Setting it in a props file was too early to make a decision.